doc/installation.md

```diff
@@ -184,6 +184,7 @@ packages=com.navercorp
 - To change the max message size of gRPC in agent, change the setting below.
 - `-Dscavenger.max-message-size`: 10MB by default
     - Supports `B`, `KB`, `MB`, `GB`, and `TB` units.
+    - Note that the Collector configured in `scavenger.conf` must also accept messages of the same size or larger.
```

Comment:
Essentially `-Dscavenger.max-message-size` won't do anything unless `-Darmeria.max-message-size` is also configured - thought it would be useful to make that clear